fw/http_parser.c

```diff
@@ -8029,6 +8042,14 @@ __h2_req_parse_cookie(TfwHttpMsg *hm, unsigned char *data, size_t len, bool fin)
 	__FSM_START(parser->_i_st);
 
 	__FSM_STATE(Req_I_CookieStart) {
+		HTTP_JA5H_REQ_CALC_NUM(((TfwHttpReq *)hm), cookie,
+				       TFW_HTTP_JA5H_COOKIE_MAX, 1);
+		cookie_num =
```

Comment:
Agreed with solution, bit in case when chunked cookie received, this will not work, you use local variable, that will be set to zero on each cookie chunk, therefore `cstate` will have wrong value. It would be better to use `parser->cstate.cookie.cookie_num` directly or for local var initialization(In this case you can declare `cookie_num` inside `__FSM_STATE(Req_I_CookieStart)` state's scope).